src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,24 @@
-export const PostInfo = () => <>Put the post here</>;
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post: { title, user, body, comments } }) => (
+  <>
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+        <UserInfo user={user} />
+      </div>
+
+      <p className="PostInfo__body">{body}</p>
+
+      <hr />
+
+      {comments.length > 0 ? (
```

Comment:
```suggestion
      {!!comments.length  ? (
```

Reply:
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/89810354/3a95fd23-5c1a-474d-87fe-65b9bc80692d)
All tests passed, but linter is indignant :) And after commit linter deleted this simbols (!!) npm test -> all tests passed again, i am little bit confused!!
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/89810354/0bec08db-4f53-43c7-8d20-b315f1bbff30)